Challange-1-iOS/Challange-1-iOS/Presentation/AvatarsListScreen/AvatarsListViewController.swift

```diff
@@ -75,6 +75,14 @@ extension AvatarsListViewController: UICollectionViewDataSource, UICollectionVie
         alert.addAction(UIAlertAction(title: "Cancel", style: .default))
         alert.addAction(UIAlertAction(title: "Delete", style: .destructive, handler: { (_: UIAlertAction) in
```

Comment:
@DanielSilva20 this logic should be moved to an outer method. Please check with @renatasiqueira , she did the same refactor recently.

Reply:
Solved